lib/asn1/der_copy.c

```diff
@@ -122,7 +124,9 @@ der_copy_bmp_string (const heim_bmp_string *from, heim_bmp_string *to)
     to->data   = malloc(to->length * sizeof(to->data[0]));
     if(to->length != 0 && to->data == NULL)
 	return ENOMEM;
-    memcpy(to->data, from->data, to->length * sizeof(to->data[0]));
+    if (to->data != NULL && from->data != NULL) {
```

Comment:
It's on the caller to not pass in a NULL `from`.

Reply:
Also, `to->data` has already been checked, thus this adds dead code for coverity to complain about.